api/representation.go

```diff
@@ -216,10 +216,10 @@ func (tr TaskRequest) String() string {
 
 type TasksResponse oapigen.TasksResponse
 
-func tasksResponseFromTaskConfigs(tcs []config.TaskConfig, requestID oapigen.RequestID) TasksResponse {
+func tasksResponseFromTaskConfigs(tcs config.TaskConfigs, requestID oapigen.RequestID) TasksResponse {
 	tasks := make([]oapigen.Task, len(tcs))
 	for i, tc := range tcs {
-		tasks[i] = oapigenTaskFromConfigTask(tc)
+		tasks[i] = oapigenTaskFromConfigTask(*tc)
```

Comment:
Nit, should this method name now be `oapigenTaskFromTaskConfig` ?

Reply:
Would you say more about why the method should be renamed now? I believe the naming is the result of the thought "`opaigen.Task` from `config.Task`". The function content and signature also hasn't really changed in this PR, so would prefer not to rename the it as part of this PR to keep it focused on TasksManager / ConditionMonitor refactoring